go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/kubernetes-sigs/aws-efs-csi-driver
 
 require (
-	github.com/aws/aws-k8s-tester/e2e/tester v0.0.0-20191204032858-d97ad2e4255e // indirect
-	github.com/aws/aws-sdk-go v1.23.21
-	github.com/container-storage-interface/spec v1.1.0
-	github.com/golang/mock v1.3.1
-	github.com/kubernetes-csi/csi-test v2.0.2-0.20190719174114-bc42f92ebdc8+incompatible
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
-	google.golang.org/grpc v1.23.0
+	github.com/aws/aws-k8s-tester/e2e/tester v0.0.0-20200527022539-8820bddbc639 // indirect
+	github.com/aws/aws-sdk-go v1.31.2
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/golang/mock v1.4.3
+	github.com/kubernetes-csi/csi-test v2.2.0+incompatible
+	github.com/onsi/ginkgo v1.12.2
+	github.com/onsi/gomega v1.10.1
+	google.golang.org/grpc v1.26.0
```

Comment:
Had to use an version prior to v1.27.0 until we pick up etcd-io/etcd#11564 (had trouble picking that commit up with go mod)